plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -337,16 +337,14 @@ def process_main(self, item=None):
         :return:     None
         """
         if isinstance(item, models.Blob):
```

Comment:
do we really need this block? since there is no need to compress here anything, we can just skip this
blob_dest_path is anyway created here further in the code https://github.com/mibanescu/pulp_docker/blob/a17b4b2738336b496f3a6195c0df83741ed2f02a/plugins/pulp_docker/plugins/importers/upload.py#L351

Reply:
I believe Manifest units go through the same function, and somehow they don't need to be renamed. You are probably right, I fail to see why they work, since we don't rename them.

Reply:
we cannot skip this entirely because blob_src_path exists and the dest path does not, so renaming those would be a correct thing to do